Dockerfile

```diff
@@ -39,31 +36,158 @@ RUN conda install --name python2 --quiet --yes \
 # https://github.com/damianavila/RISE
 RUN conda install --quiet --yes -c damianavila82 rise
 
+# install zeroc-ice and python-omero
 RUN conda install --name python2 --quiet --yes -c bioconda zeroc-ice && \
-    conda install --name python2 --quiet --yes -c pdrops pygraphviz && \
-    /opt/conda/envs/python2/bin/pip install \
+    conda install --name python2 --quiet --yes -c bioconda python-omero=5.3.3
+
+# install idr-py and notebook dependencies
+RUN /opt/conda/envs/python2/bin/pip install \
         graphviz \
         gseapy \
         py2cytoscape \
         pydot \
         tqdm \
-        idr-py==0.1
-
-# Display resource usage in notebooks https://github.com/yuvipanda/nbresuse
-RUN pip install https://github.com/IDR/nbresuse/archive/0.1.0-idr.zip && \
```

Comment:
Why did you remove `nbresuse`?

Reply:
I had installation issues at this step, while I was working with the Non-IDR version of this docker file for the Cambridge training (because of the base Image I had chosen, happy to add it back and check what the issues were)

Reply:
For purposes of review it might be easier to split this PR:
1. minimal image with omero bioconda
2: CellProfiler
3: R
?

Reply:
Happy to do that, if that makes more sense.